plugins/easyimage/styles/easyimage.css

```diff
@@ -4,7 +4,6 @@ For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
 */
 
 .easyimage, .cke_widget_wrapper_easyimage {
-	background-color: #fff;
```

Comment:
Probably better approach would be just explicitly setting background to `transparent`, without resetting styles for `figure`.